docs/web-content/website-content-and-presentation.md

```
@@ -0,0 +1,746 @@
+        font-size: 11px;
+    }
+}
+```
+
+---
+
+## Headings, landmarks, and sections
+
+Use **one `<h1>`** per page. Increase heading level **by one** per nesting depth (`h1` → `h2` → `h3`). Headings are **signposts**, not slogans.
+
+#### Do
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)